ops/pebble.py

```diff
@@ -181,6 +182,19 @@
     total=False,
 )
 
+LocalIdentityDict = typing.TypedDict('LocalIdentityDict', {'user-id': int})
+BasicIdentityDict = typing.TypedDict('BasicIdentityDict', {'password': str})
+
+IdentityDict = typing.TypedDict(
```

Comment:
This is a bit confusing, could we have better names?

I made the mistake of thinking that `IdentityDict` is like a base class for `LocalIdentityDict`. I imagine some charmers may make the same mistake.

Reply:
I think the existing names are okay, as they match the [Pebble Go client's naming](https://pkg.go.dev/github.com/canonical/pebble@v1.20.0/client#Identity): `Identity` is the "root", and then we have `LocalIdentity` and `BasicIdentity` (plus the "Dict" suffix for the plain dict types).

Reply:
As discussed in the daily, this has been resolved.